publics/ext.js

```diff
@@ -1,205 +1,204 @@
-    var form = document.getElementById("itemForm");
-    //form.onsubmit = onFormSubmit;
-    //form.addEventListener("submit",onFormSubmit);
-    document.getElementById("fEmail").onchange = function(){validateData(form,true)};
-    document.getElementById("fPhone").onchange = function(){validateData(form,true)};
-
-    var rqst = new XMLHttpRequest();
-    rqst.open("GET", "/items", true);
-    rqst.onreadystatechange = onRequest;
-    rqst.send();
-
-    function onRequest(){
-        if (this.readyState != 4)
-            return;
-        if (this.status != 200) {
-            return;
-        } else {
-            var items = JSON.parse(this.responseText);
-            if (!!items) {
-                refreshItems(items);
-            }
+'use strict';
```

Comment:
re-fixed in https://github.com/crushcat/epam-tasks-app2/pull/2/commits/87a3aea2702d6ed38d4f3a6b72de4d15d263b9fa